props/BC_Working_temp_relevant_only.h

```diff
@@ -76,103 +202,123 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
       SetMute(true);
     }
   }
+// more stuff
+
+
 
 #ifdef BC_DUAL_BLADES
 
   static const BladeSet controlled_blades_;
 
-  void PrintBladesStatus() {
-    PVLOG_DEBUG << "***** Updated. Current blade on: " 
-           << (SaberBase::OnBlades()[BC_MAIN_BLADE] ? " MAIN " : "") 
-           << (SaberBase::OnBlades()[BC_SECOND_BLADE] ? " SECOND " : "") 
-           << "\n";
-  }
-
+  
   void TurnBladeOn(BladeSet target_blade) {  // receives opposite blade (MAIN or SECOND) so we can exclude it
```

Comment:
Is this comment still accurate?
If so, you should fix that.
If not, remove the comment.

Reply:
I would recommend doing this:

// Add all non-controlled blades.
target_blade = target_blade | ~controlled_blades_;

That way you don't have to feed in the "opposite blade" anymore.